internal/output/outputter.go

```diff
@@ -88,9 +88,9 @@ func (o *Outputter) StartBuildStep(format string, a ...interface{}) {
 	o.curBuildStep++
 }
 
-func (o *Outputter) Print(format string, a ...interface{}) {
+func (o *Outputter) Printf(format string, a ...interface{}) {
```

Comment:
go vet has magic where it will check that the format string has the right number of arguments if you call the function "Printf"

Reply:
Neat! Good to know.